components/widget/match/summary/ffa/widget_match_summary_ffa_points_distribution.lua

```diff
@@ -0,0 +1,64 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Widget/Match/Summary/Ffa/PointsDistribution
```

Comment:
fwiw some wikis don't have "kills" but other means of getting points
so we can not use this (in its current form) for those wikis
(example: sc2 has various different ffa modes some where you obtain points by defending a certain part of the map (usually the middle) for a given time)

Reply:
We'll sort it out when we get there, make it more flexible :)